src/shipit_pulse_listener/default.nix

```diff
@@ -24,6 +24,29 @@ EOF
     '';
   });
 
+  mkBot = branch:
+    let
+      secretsKey = "repo:github.com/mozilla-releng/services:branch:" + branch;
+    in
+      mkTaskclusterHook {
+        name = "Triggers Taskcluster hooks on pulse messages";
+        owner = "babadie@mozilla.com";
+        schedule = [ "0 0 * * * *" ];  # every hour
```

Comment:
No, messages are treated right away in the asyncio event loop.
This only means that the task is re-created every hour.